source/boards/focalboard/installing-boards.rst

```diff
@@ -0,0 +1,376 @@
+Installing the Focalboard App
```

Comment:
@justinegeffen This page is missing from the TOC https://docs.mattermost.com/guides/boards.html#access-your-workspace

Reply:
Thanks, Amy - oversight on my part! Fixing now.